packages/components/src/LoadingSpinner.scss

```diff
@@ -1,6 +1,42 @@
+.loading-spinner {
+  --primary-color: var(--dh-accent-color, #4c7dee);
+  /* stylelint-disable-next-line alpha-value-notation */
+  --secondary-color: rgba(240, 240, 240, 0.5);
```

Comment:
This needs to be based on a variable. A hardcoded value here would look terrible on a light theme for example.

Reply:
Any thoughts on what it should be tied to?

Reply:
I'll tie it to a semantic color for now and refine it once I get into the color mapping